src/module/item/base/sheet/rule-element-form/grant-item.ts

```diff
@@ -7,7 +7,7 @@ class GrantItemForm extends RuleElementForm<GrantItemSource, GrantItemRuleElemen
     override async getData(): Promise<GrantItemFormSheetData> {
         const data = await super.getData();
         const uuid = this.rule.uuid ? String(this.rule.uuid) : null;
-        const granted = uuid ? await fromUuid(uuid) : null;
+        const granted = uuid && !uuid.startsWith("{") ? await fromUuid(uuid) : null;
```

Comment:
use `UUIDUtils.isItemUUID`

Reply:
Oh right, there was a helper for that. Updated.